src/idsk/components/subscription-form/template.njk

```diff
@@ -18,17 +20,17 @@ Tento súhlas môžem kedykoľvek odvolať na <a href="mailto:info@slovenskoit.s
     <span class="idsk-subscription-form__confirm-text">{{ confirmDescription | safe }}</span>
   </p>
 
-  <div class="govuk-grid-row">
-    <form action="{{ params.formAction if params.formAction else "#" }}"
-          class="govuk-form-group {{ params.targetClass if params.targetClass else "idsk-subscription-form__submit-handler" }}">
-      <div class="govuk-grid-column-three-quarters">
-        <input type="email" class="govuk-input" title="{{ inputText }}">
-      </div>
-      <div class="govuk-grid-column-one-quarter">
-        <input type="submit" class="idsk-button" value="{{ buttonText }}">
-      </div>
-    </form>
-  </div>
+  <form action="{{ params.formAction if params.formAction else "#" }}"
+        class="govuk-form-group idsk-subscription-form__input {{ params.targetClass if params.targetClass else "idsk-subscription-form__submit-handler" }}">
+    {{ govukInput({
+      "label": {
+        "text": inputText
+      },
+      "id": "input-example",
```

Comment:
Toto id by som pozmenil, pokial pouzivatel nema moznost to cez nunjuck premennu zmenit tak by som to nenazyval example...